attachment_log/models/ir_attachment.py

```diff
@@ -0,0 +1,52 @@
+from odoo import _, models
+from odoo.fields import Datetime
+
+
+class IrAttachment(models.Model):
+    _inherit = "ir.attachment"
+
+    def _send_attachment_notification(self, is_create=False, is_unlink=False):
```

Comment:
Posting messages in chatter should be an option that is toggled in the configuration. Because some users might considered it not useful and even annoying.